spec/models/task_spec.rb

```diff
@@ -237,6 +248,28 @@
                                                            'task_id'))
                                                        end)
     end
+
+    context 'when task is cleanly duplicated' do
+      subject(:clean_duplicate) { task.clean_duplicate user }
+
+      let(:user) { create(:user) }
+
+      it 'has the current user' do
+        expect(clean_duplicate.user).to be user
+      end
+
+      it 'is private' do
+        expect(clean_duplicate.access_level).to eq(:private.to_s)
+      end
+
+      it 'is not part of a group' do
+        expect(clean_duplicate.groups).to eq([])
+      end
+
+      it 'has a modified title' do
+        expect(clean_duplicate.title).to eq("#{I18n.t('tasks.copy_of_task')}: #{task.title}")
```

Comment:
This actually resembles the implementation very much. In this case, I would actually suggest having a static string defined for the expected way. Otherwise, we might miss some errors in the future.

Reply:
You're right. I have made the changes.